pointsource_imaging/NIRCam_Basic_PSF_Photometry_solution.ipynb

```diff
@@ -74,7 +74,7 @@
    "id": "95891849",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from collections import OrderedDict
</pre><p>Please move this import to the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='116'/>